
<frozen importlib._bootstrap>:1047: ImportWarning: ImportHookFinder.find_spec() not found; falling back to find_module()
[1m============================================================ test session starts =============================================================
platform linux -- Python 3.11.7, pytest-7.1.2, pluggy-1.3.0 -- /home/tta/miniconda3/envs/mlflow-fe2587c81cc6688bca065c82817d4d8591227f1d/bin/python
cachedir: .pytest_cache
rootdir: /app/udacity/mlops/genre_classification/check_data
[1mcollected 4 items                                                                                                                            
test_data.py::test_column_presence_and_type [32mPASSED
test_data.py::test_class_names [32mPASSED
test_data.py::test_column_ranges [32mPASSED
test_data.py::test_kolmogorov_smirnov [32mPASSED
[33m============================================================== warnings summary ==============================================================
../../../../../home/tta/miniconda3/envs/mlflow-fe2587c81cc6688bca065c82817d4d8591227f1d/lib/python3.11/site-packages/wandb/apis/public.py:3111
  /home/tta/miniconda3/envs/mlflow-fe2587c81cc6688bca065c82817d4d8591227f1d/lib/python3.11/site-packages/wandb/apis/public.py:3111: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    from pkg_resources import parse_version
../../../../../home/tta/miniconda3/envs/mlflow-fe2587c81cc6688bca065c82817d4d8591227f1d/lib/python3.11/site-packages/pkg_resources/__init__.py:2871
  /home/tta/miniconda3/envs/mlflow-fe2587c81cc6688bca065c82817d4d8591227f1d/lib/python3.11/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('google')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)
test_data.py::test_column_presence_and_type
  /app/udacity/mlops/genre_classification/check_data/conftest.py:29: DtypeWarning: Columns (16) have mixed types. Specify dtype option on import or set low_memory=False.
    sample1 = pd.read_csv(local_path)
test_data.py::test_column_presence_and_type
  /app/udacity/mlops/genre_classification/check_data/conftest.py:32: DtypeWarning: Columns (16) have mixed types. Specify dtype option on import or set low_memory=False.
    sample2 = pd.read_csv(local_path)
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
[33m======================================================= [32m4 passed[39m, [33m[1m4 warnings[22m in 4.63s ========================================================